Classes/Issues/IssueManagingContextController.swift

```diff
@@ -363,10 +399,24 @@ final class IssueManagingContextController: NSObject, ContextMenuDelegate {
         } else if let people = viewController as? PeopleViewController {
             didDismiss(controller: people)
         } else if let labels = viewController as? LabelsViewController {
-            didDismiss(selected: labels.selected)
+            didDismiss(controller: labels)
         }
     }
 
     func contextMenuDidDismiss(viewController: UIViewController, animated: Bool) {}
 
+    func showCancelAlert(onKeep: @escaping () -> Void) {
+        guard let viewController = self.viewController else { return }
+
+        let title = NSLocalizedString("Are you sure?", comment: "")
+        let message = NSLocalizedString("Some changes have been made. Are you sure to discard them?", comment: "")
```

Comment:
`Are you sure you want to discard them?` might be a bit more pleasing to read, I think? :)

Reply:
Done